docs/dev/build-instructions/build_linux.md

```diff
@@ -160,6 +160,15 @@ For all available build options, have a look [here](cmake_options.md).
 E.g. if you only want to build the server, use `cmake -Dclient=OFF ..`.
 
 
+## Using the BUILD_NUMBER variable
```

Comment:
This is not Linux specific. Hence, I think this should go into the general instructions